tools/shell-client.ts

```diff
@@ -22,18 +22,14 @@ export function connect(shellDir: string) {
 }
 
 class Client {
-  shellDir: string;
-  exitOnClose = false;
-  firstTimeConnecting = true;
-  connected = false;
-  reconnectCount = 0;
-  reconnectTimer?: NodeJS.Timeout;
+  public connected = false;
 
-  constructor(shellDir: string) {
-    assert.ok(this instanceof Client);
+  private exitOnClose = false;
+  private firstTimeConnecting = true;
+  private reconnectCount = 0;
+  private reconnectTimer?: NodeJS.Timeout;
 
-    this.shellDir = shellDir;
-  }
+  constructor(public shellDir: string) {}
```

Comment:
One of my favorite TypeScript features: you can declare instance variables with appropriate visibility and types right in the parameter list of the constructor!

Reply:
Parameter properties)

Could you please provide a feedback in https://forums.meteor.com/t/request-typescript-code-style/49498 about Code Style?